### PR TITLE
Bugfix(frontend): remove unstyled initial render

### PR DESCRIPTION
**Pull request by topocount**
We're now utilizing MaterialUI's Server-side styling library to collect
and pre-render the needed css for  the initial loading screen.

test plan:
1. yarn start

2. navigate to http://localhost:8080 and disable javascript. Reload the
page and ensure it's styled

3. Reenable javascript, reload the page and ensure everything loads
fully as expected

### CODE REVIEW

File: src/ui/server.js [hunk 1]
```diff
@@ -22,25 +22,30 @@ export default function render(
...
       <head>
       <meta charset="utf-8">
       <meta name="viewport" content="width=device-width,initial-scale=1" />
       <link rel="shortcut icon" href="${assets.resolve("/favicon.png")}" />
-      <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
-      <link href="https://fonts.googleapis.com/css?family=Roboto+Condensed" rel="stylesheet">
       <title>SourceCred</title>
       <style>${require("./index.css")}</style>
-      <style data-aphrodite>${css.content}</style>
+      <style id="jss-server-side">${css}</style>
```
Comment (wchargin) [inline src/ui/server.js:48]: We still use Aphrodite, so let’s keep both? I expect that it should be
possible to nest the callbacks.

**Review by wchargin — COMMENTED** [review]
Lovely! This is what I was hoping the patch would look like.

File: src/ui/server.js [hunk 2]
```diff
@@ -22,25 +23,31 @@ export default function render(
...
+
     const page = dedent`\
       <!DOCTYPE html>
       <html>
       <head>
       <meta charset="utf-8">
       <meta name="viewport" content="width=device-width,initial-scale=1" />
       <link rel="shortcut icon" href="${assets.resolve("/favicon.png")}" />
-      <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
-      <link href="https://fonts.googleapis.com/css?family=Roboto+Condensed" rel="stylesheet">
```
Comment (wchargin) [inline src/ui/server.js:40]: Is this related? I am happy to drop these, but should we move that to a
different PR?

File: yarn.lock [hunk 3]
```diff
@@ -7261,6 +7261,11 @@ nan@^2.12.1:
   resolved "https://registry.yarnpkg.com/nan/-/nan-2.14.1.tgz#d7be34dfa3105b91494c3147089315eff8874b01"
   integrity sha512-isWHgVjnFjh2x2yuJ/tj3JbwoHu3UC2dX5G/88Cm24yB6YopVgxvBObDY7n5xW6ExmFhJpSEQqFPvq9zaXc8Jw==
 
+nan@^2.14.0:
```
Comment (wchargin) [inline yarn.lock:7264]: Why are we changing the lockfile when `package.json` is unchanged?

**Review by wchargin — COMMENTED** [review]
I can’t test this because of #2531, but the patch looks good to me
modulo inlines.

File: src/ui/server.js (hunk 1 above)
Comment (topocount) [reply src/ui/server.js]: Yup! having the nested callbacks works fine

File: src/ui/server.js (hunk 2 above)
Comment (topocount) [reply src/ui/server.js]: These lines have been un-removed from the pull request

File: yarn.lock (hunk 3 above)
Comment (topocount) [reply yarn.lock]: This is no longer necessary after the reversion

**Review by decentralion — APPROVED** [review]

**Review by wchargin — COMMENTED** [review]
post-hoc LGTM; thanks!
